Event/RefreshEvent.php

```diff
@@ -12,27 +12,41 @@
 namespace Gesdinet\JWTRefreshTokenBundle\Event;
 
 use Gesdinet\JWTRefreshTokenBundle\Model\RefreshTokenInterface;
-use Symfony\Component\Security\Guard\Token\PostAuthenticationGuardToken;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 
 class RefreshEvent extends Event
 {
+    /**
+     * @var RefreshTokenInterface
+     */
     private $refreshToken;
 
-    private $preAuthenticatedToken;
+    /**
+     * @var TokenInterface
+     */
+    private $token;
 
-    public function __construct(RefreshTokenInterface $refreshToken, PostAuthenticationGuardToken $preAuthenticatedToken)
+    public function __construct(RefreshTokenInterface $refreshToken, TokenInterface $token)
```

Comment:
This is a type widening change, only the `Symfony\Component\Security\Guard\Token\PostAuthenticationGuardToken` token was accepted but with this PR the interface is allowed.